### ISSUE TITLE
Truffle Debugger 

### ISSUE THREAD

**Issue by Emiller88** [ethereum/emacs-solidity#22]
It would be cool if we could get some integration for the new truffle debugger. I'm not sure if that's possible or not.
[Truffle Debugging](http://truffleframework.com/docs/getting_started/debugging)

**Comment by LefterisJP** [comment]
Yeah would be a nice thing to have but I don't have the time to look into it right now. Will keep the issue open and see when I can find time to do it.

In the meantime this is an open source project so if anyone has time to look into it and open a PR that would be awesome!

**Comment by Emiller88** [comment]
Do you have any idea where to start? I'm pretty new to emacs but would love to take a stab at it.

**Comment by LefterisJP** [comment]
@Emiller88 Sorry for the very late reply. I don't have any pointers on truffle as I have not used it much myself. For emacs lisp you can check any tutorial you can find in the internet for example [this](https://github.com/caiorss/Emacs-Elisp-Programming/blob/master/Elisp_Programming.org) .. and then armed with that knowledge read and understand the code of the emacs solidity module.
